addons/hw_posbox_homepage/views/handler_list.html

```diff
@@ -23,18 +23,122 @@
+                                <option value="{{ log_level }}"
+                                        {% if (interface_logger_info.is_using_parent_level == False) and (log_level == interface_logger_info.level) %}
+                                        selected
+                                        {% endif %}
+                                >
+                                    {{ log_level | capitalize }}
+                                </option>
+                                {% endfor %}
+                            </select>
+                        {% else %}
+                            <span class="font-small">Logger uninitialised</span>
+                        {% endif %}
+                    </td>
+                </tr>
+            {%- endfor %}
+        </table>
+
+        <h2 class="text-center text-green">Drivers list</h2>
+        <table align="center" width="80%" cellpadding="3">
+            <tr>
+                <th>Name</th>
+                <th>Log Level</th>
+            </tr>
+            {% for driver in drivers_list -%}
+                <tr>
+                    <td>{{ driver }}</td>
+                    <td>
+                        {% set driver_logger_info = drivers_logger_info[driver] %}
+                        {% if driver_logger_info != False %}
+                            <select name="iot-logging-driver-{{driver}}">
+                                <option value="parent"
+                                    {% if driver_logger_info.is_using_parent_level == True %}
+                                        selected
+                                    {% endif %}
+                                >
+                                    Same as {{ driver_logger_info.parent_name | capitalize }} ({{driver_logger_info.level | capitalize}})
+                                </option>
+                                <option style="font-size: 1pt; background-color: #000000;" disabled>&nbsp;</option>
+                                {% for log_level in available_log_levels %}
+                                <option value="{{ log_level }}"
+                                        {% if (driver_logger_info.is_using_parent_level == False) and (log_level == driver_logger_info.level) %}
+                                        selected
+                                        {% endif %}
+                                >
+                                    {{ log_level | capitalize }}
+                                </option>
+                                {% endfor %}
+                            </select>
+                        {% else %}
+                            <span class="font-small">Logger uninitialised</span>
+                        {% endif %}
+                    </td>
+                </tr>
+            {%- endfor %}
+        </table>
+
+    <input class="btn" type="submit" value="Update Logs Level Value"/>
+    </form>
     {% if server %}
```

Comment:
<div style="margin-top: 20px;" class="text-center">     
          <input class="btn" type="submit" value="Update Logs Level Value"/>
      </form>   
{% if server %}
        <div style="margin-top: 20px;" class="text-center">
            <a id="load_handler_btn" class="btn" href='/load_iot_handlers'>Load handlers</a>
        </div>
        <div class="text-center font-small" style="margin: 10px auto;">
            You can clear the handlers configuration
            <form style="display: inline-block;margin-left: 4px;" action='/handlers_clear'>
                <input class="btn btn-sm" type="submit" value="Clear"/>
            </form>
        </div>
    {% endif %}

Reply:
![image](https://github.com/odoo/odoo/assets/60775325/e80813cb-bce8-49db-aae9-1c1616901ad9)